### ISSUE TITLE
enhancement: default cssClassName?

### ISSUE THREAD

**Issue by tony-kerz** [GriddleGriddle/Griddle#309]
any interest in setting up a default class name for columns?

whenever i specify class-name (so far for things like width or whitespace), 
i'm pretty much just typing in what the result of `_.kebabCase(columnName)` would be...

could just be an opportunity for some convention-over-configuration code thinning...

**Comment by ryanlanciaux** [comment]
That sounds like a good idea. So, to confirm if i had a column called firstName it would automatically apply the css class name 'first-name' to the column?

**Comment by tony-kerz** [comment]
yep, `columnName: 'firstName'` results in `cssClassName: 'first-name'`, but without the typing.
or maybe just allow for a function to be provided at a high level (either by passing a prop to a `Griddle` element, or some other one time setup like `Griddle.setOptions()`)
something like:

```
columnClassNameDeriver: (metadata) => { return _.kebabCase(metadata.columnName) }
```

i'm a sass user, so i can take that and do something like:
```
.people {
  table {
    th {
      white-space: nowrap;
    }
    td.birthdate, td.first-name, td.last-name, td.phone, td.address, td.city, td.state {
      white-space: nowrap;
    }
  }
}
```

**Comment by ryanlanciaux** [comment]
This sounds like something that would be an awesome plugin in 1.0. Trying to think of the best way to go about doing this.

**Comment by tony-kerz** [comment]
i love the plugin concept for 1.x.

thanks ryan. feel free to shoot me any ideas around how this particular guy may manifest as a plugin, 
i would be happy to try my hand at one.

**Comment by ryanlanciaux** [comment]
I realize that quite a bit of time has gone on since this conversation but we just released Griddle 1.0 and I've been trying my best to follow-up on open issues. The plugin API is released and [documentation is available here](http://griddlegriddle.github.io/Griddle/docs/plugins/).

For this specific feature, if there was any interest, I think that the plugin could make an enhancer for the Column component. The ColumnEnhancer would look at the column name and add the classname to the className prop. 

Closing this one for now and thank you for this suggestion. Please feel free to follow up here, gitter or open any new issues if problems are encountered. :+1:
